bin/check-topic.rb

```diff
@@ -0,0 +1,152 @@
+#! /usr/bin/env ruby
+#
+# check-topic
+#
+# DESCRIPTION:
+#   This plugin checks topic properties.
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: zookeeper
+#
+# USAGE:
+#   ./check-topic
+#
+# NOTES:
+#
+# LICENSE:
+#   Olivier Bazoud
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'json'
+require 'zookeeper'
+
+class TopicsCheck < Sensu::Plugin::Check::CLI
+  option :zookeeper,
```

Comment:
I have not really looked into `kafka` can you confirm this makes sense to live here and not in the zookeeper plugin?

Reply:
@majormoses 
It complements the zookeeper plugin

Which I've also tested
Provides more Kafka-specifics checks
Especially the one relying on the HTTP servlet
Which actually exposés Dropwizard metrics, not available through zookeeper